public/css/app.css

```diff
@@ -666,17 +670,19 @@ form span {
   color: #000;
   display: inline-block;
   width: 40vw;
-  min-width: 300px;
+  min-width: 350px;
   max-width: 600px;
   padding: calc(3.33vw + 0.625rem) !important;
   text-align: left;
   box-sizing: border-box;
+  box-shadow: 0 0 9px #0000002e;
```

Comment:
Personal question (for another issue/PR), but why sometimes `rgba()` and sometimes `#rrggbbaa` format?
https://caniuse.com/#search=rgba seems to say that `rgba()` support is in the at around 97% whereas `#rrggbbaa` support is unofficial and at around 73%.

Reply:
I am trying to think of a good answer to this question and I will let you know when I arrive at one.